File: articles/algorithmic-trust-verification-and-security/julia/algorithmic_trust_demo.jl
function trust_quality(verification, validation, security, provenance, monitoring, governance)
    return 100 * (0.18*verification + 0.18*validation + 0.18*security + 0.16*provenance + 0.15*monitoring + 0.15*governance)
end

println("trust quality=$(round(trust_quality(0.88, 0.82, 0.88, 0.90, 0.84, 0.82), digits=3))")

File: articles/algorithmic-trust-verification-and-security/rust/algorithmic_trust_demo.rs
fn trust_quality(verification: f64, validation: f64, security: f64, provenance: f64, monitoring: f64, governance: f64) -> f64 {
    100.0 * (0.18 * verification + 0.18 * validation + 0.18 * security + 0.16 * provenance + 0.15 * monitoring + 0.15 * governance)
}

fn main() {
    println!("trust quality={:.3}", trust_quality(0.88, 0.82, 0.88, 0.90, 0.84, 0.82));
}
